scipy/stats/_continuous_distns.py

```diff
@@ -1846,6 +1847,109 @@ def _entropy(self, c):
 weibull_min = weibull_min_gen(a=0.0, name='weibull_min')
 
 
+class truncweibull_min_gen(rv_continuous):
+    r"""A truncated Weibull minimum continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `truncweibull_min` is:
+
+    .. math::
+
+        f(x, a) = c x^{c-1} \exp(-x^c)
+
+    for :math:`x > a`, :math:`c > 0`.
+
+    `truncweibull_min` takes :math:`a` and :math:`b` as shape parameters.
+
+    Notice that the truncation value, :math:`a`, is defined in standardized
+    form::
+
+        a = (u - loc)/scale
+
+    where :math:`u` is the specific truncation value.
+
+    %(after_notes)s
+
+    %(example)s
+
+    """
+    def _argcheck(self, a, c):
+        self.a = a
+        return (a >= 0.) & (c > 0.)
+
+    def _pdf(self, x, a, c):
+        return c*pow(x, c-1)*np.exp(-pow(x, c) + pow(a, c))
+
+    def _logpdf(self, x, a, c):
+        logpdf = np.log(c) + sc.xlogy(c - 1, x) - pow(x, c) + pow(a, c)
+        return logpdf
+
+    def _cdf(self, x, a, c):
+        return -sc.expm1(-pow(x, c) + pow(a, c))
+
+    def _sf(self, x, a, c):
+        return np.exp(-pow(x, c) + pow(a, c))
+
+    def _logsf(self, x, a, c):
+        return -pow(x, c) + pow(a, c)
+
+    def _ppf(self, q, a, c):
+        return pow(pow(a, c) - sc.log1p(-q), 1.0/c)
```

Comment:
Since Python 2 is dropped you can just write `return (a**c - sc.log1p(-q))**(1/c)`